### ISSUE TITLE
Some images from imported board not showing while present in file system - Nginx FASTCGI problem?

### ISSUE THREAD

**Issue by darkocoit** [RestyaPlatform/board#3725]
Hi guys.
I've been using and debugging restya for a while (lost access to my old account)
We have imported aprox 20 large boards from Trello.

Now, when I try to open some of the images (not all) I only get an image stating: The requested resource cant be loaded.
I searched for the specific images in /usr/share/nginx/html/restyaboard/media/card/xxx and the image is there, I can open it without problems.
The nginx error log states this:

2019/12/05 11:57:41 [error] 5969#5969: *129041 FastCGI sent in stderr: "PHP message: PHP Warning:  copy(/usr/share/nginx/html/restyaboard/media/media/Card/108/Potpornenoge.jpg): failed to open stream: No such file or directory in /usr/share/nginx/html/restyaboard/server/php/image.php on line 188" while reading response header from upstream, client: 10.10.10.5, server: boards.tranexp.com, request: "GET /img/original/CardAttachment/394.100536108dbb60580d89d57bb74e2d70.png HTTP/1.0", upstream: "fastcgi://unix:/run/php/php7.0-fpm.sock:", host: "boards.tranexp.com"

I'm using Restyaboard 0.6.7.

**Comment by rrjanbiah** [comment]
`image.php` through nginx rules will copy images from `media` folder to `img` folder... during the thumbnail generation process. Please double check if you have write permission for `img` folder

**Comment by darkocoit** [comment]
Most images open perfectly. I think this is the problem:

PHP message: PHP Warning: copy(/usr/share/nginx/html/restyaboard**/media/media/**Card/108/Potpornenoge.jpg): 

I think that the problem is in the syntax because folder /media has two levels (media/media/Card) instead of only one. I solved it temporarily by copying all files to a newly created subfolder "media" like the syntax says and the images are loading.

Where can I correct the above syntax?:

**Comment by sridhar391** [comment]
@darkocoit We have resolved this issue in the `0.6.7`  by removing the hard coded path in the databse by executing the below SQL query in [upgrade-0.6.6-0.6.7.sql](https://github.com/RestyaPlatform/board/blob/dev/sql/upgrade-0.6.6-0.6.7.sql) and we have  the folder configurations inside the `config.inc.php` file in line 29 like https://github.com/RestyaPlatform/board/blob/master/server/php/config.inc.php#L29

```
UPDATE users SET profile_picture_path = REPLACE(profile_picture_path, 'media/', '');
UPDATE card_attachments SET path = REPLACE(path, 'media/', '');
UPDATE card_attachments SET doc_image_path = REPLACE(doc_image_path, '/img/', '');
UPDATE boards SET background_picture_path = REPLACE(background_picture_path, 'media/', '');
UPDATE organizations SET logo_url = REPLACE(logo_url, 'media/', '');
```
